packages/flutter/lib/src/material/outline_button.dart

```diff
@@ -303,33 +323,30 @@ class _OutlineButtonState extends State<OutlineButton> with SingleTickerProvider
 
   // TODO(hmuller): this method is the same as FlatButton
   Color _getTextColor(ThemeData theme, ButtonThemeData buttonTheme) {
-    final Color color = widget.enabled ? widget.textColor : widget.disabledTextColor;
-    if (color != null)
-      return color;
+    final Color color =
+        widget.enabled ? widget.textColor : widget.disabledTextColor;
+    if (color != null) return color;
```

Comment:
this is one example where dartfmt just doesn't apply our style guide at all so you'll have to manually fix it, either by adding braces around the block (and rerunning dartfmt) or by adding a manual newline.

Reply:
Done